.github/workflows/main.yml

```diff
@@ -15,17 +15,17 @@ jobs:
         uses: actions/checkout@v4
 
       - name: Build Docker Container
-        run: docker compose -f docker-compose.yml build >/dev/null
+        run: docker compose -f docker-compose.yml build
```

Comment:
I don't see why we would want to hide the build output - this could be very useful to debug build failures.